src/components/Comments/ContainerComment.js

```diff
@@ -138,6 +140,7 @@ export const ReplyComment = ({
               <TouchableWithoutFeedback onLongPress={() => onCommentLongPressed(item, 1)}>
                 <View key={`c${index}`} style={styles.levelOneCommentWrapper}>
                   <Comment
+                    feedId={feedId}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Adding `feedId` as a prop to the `Comment` component is acceptable. Ensure that this new prop is being correctly passed down from parent components and used appropriately within the `Comment` component.
```diff
+                    feedId={feedId}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->